### PR TITLE
CODE RUB: Updated variable in retrieve all Calendar Entry

### PR DESCRIPTION
**Pull request by deepshri7**

### CODE REVIEW

**Comment by hassanhabib** [comment]
@deepshri7 fix the conflicts here.
Also separate each task in it's own PR.

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs [hunk 1]
```diff
@@ -24,15 +24,15 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllCalendarEntriesWhenSqlExc
...
             // when
             Action retrieveAllCalendarEntryAction = () =>
                 this.calendarEntryService.RetrieveAllCalendarEntries();
-           
+
             // then
             Assert.Throws<CalendarEntryDependencyException>(
                 retrieveAllCalendarEntryAction);
-
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs:35]: run code cleanup

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs [hunk 2]
```diff
@@ -61,12 +61,12 @@ public void ShouldThrowServiceExceptionOnRetrieveAllCalendarEntriesWhenException
                     .Throws(serviceException);
 
             // when
-            Action retrieveAllCalendarEntryAction = () =>
-                this.calendarEntryService.RetrieveAllCalendarEntries();
+           Action retrieveAllCalendarEntryAction = () =>
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs:64]: fix this

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs (hunk 1 above)
Comment (ShriHumrudha) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs:35]: @deepshri7 this is pending

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs (hunk 2 above)
Comment (deepshri7) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs]: what change in this ?

Comment (deepshri7) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs]: no changes

**Comment by ShriHumrudha** [comment]
Difference is only extra tabs

**Pull request closed without merging**